content/blog/usemodels-0-0-1/index.Rmd

```diff
@@ -0,0 +1,106 @@
+---
+output: hugodown::hugo_document
+
+slug: usemodels-0-0-1
+title: usemodels 0-0-1
+date: 2020-09-29
+author: Max Kuhn
+description: >
+    The new usemodels R package is a helpful way to automatically generate 
+    tidymodels code. 
+
+photo:
+  url: https://unsplash.com/photos/0TH1H1rq_eY
+  author: Neven Krcmarek
+
+categories: [package] 
+tags: [tidymodels,parsnip,recipes,tune]
+---
+
+```{r startup, include = FALSE}
+library(tidymodels)
+library(usemodels)
+options(width = 90)
+```
+
+We're very excited to announce the first release of the [usemodels](https://usemodels.tidymodels.org/) package. usemodels creates templates for tidymodels analyses so you don't have to write as much new code. 
```

Comment:
```suggestion
We're very excited to announce the first release of the [usemodels](https://usemodels.tidymodels.org/) package. The tidymodels packages are designed to provide modeling functions that are highly flexible and modular. This is powerful but sometimes a template or skeleton showing how to start is helpful. The usemodels package creates templates for tidymodels analyses so you don't have to write as much new code. 
```

Reply:
Like the change, would just suggest a comma after powerful in the second to last sentence (I can't suggest on a suggestion)